doc/deprecated.md

```diff
@@ -109,6 +109,22 @@ info, see [#7463] and [#5976].
 
 </details>
 
+<details>
+<summary>2024-12-31: retiring legacy Pubsub admin APIs</summary>
+<br>
+
+- On 2024-12-31 (or shortly after) we are planning to remove the hand-written
```

Comment:
Perhaps this should use the more intentional _"We will remove ..."_ text, like above.

Reply:
done